integrations/tensorflow/iree_tf_compiler/TF/LowerGlobalTensors.cpp

```diff
@@ -179,7 +179,7 @@ void LowerGlobalTensors::runOnOperation() {
           *latticeElement->getValue().ops.begin();
       ml_program::GlobalOp globalOp = symbolRefMap[globalTensor];
 
-      for (Operation *user : val.getUsers()) {
+      for (Operation *user : llvm::make_early_inc_range(val.getUsers())) {
```

Comment:
Can you add a test case that runs on presubmit for this?

Reply:
Good idea, http://github.com/iree-org/iree/pull/9982, only triggers with ASAN or TSAN but I believe those are now on presubmit.